modules/oe_content_call_proposals/config/install/field.storage.node.oe_call_proposals_model.yml

```diff
@@ -0,0 +1,32 @@
+langcode: en
+status: true
+dependencies:
+  module:
+    - node
+    - options
+id: node.oe_call_proposals_model
+field_name: oe_call_proposals_model
+entity_type: node
+type: list_string
+settings:
+  allowed_values:
```

Comment:
I would propose to implement `allowed_values_function:  'oe_content_call_proposals_model_allowed_values'`.

```
function oe_content_call_proposals_model_allowed_values() {
  return [
    CallForProposalsNodeWrapperInterface::MODEL_SINGLE_STAGE => 'Single-stage',
    CallForProposalsNodeWrapperInterface::MODEL_TWO_STAGE => 'Two-stage',
    // etc.
  ];
}
```
Using this approach it will be easier to add new models to the node.oe_call_proposals_model field or rearrange items if needed. Also no need to hardcode model values in the code and we can process models in CallForProposalsNodeWrapper like we work with statuses. 

Looks like such approach should be discussed and approved since `allowed_values` is used now in another fields. But such approach can be useful for Blending Project as well.

@drishu what do you think?

Reply:
Excellent idea, I implemented the changes.

Reply:
yes allowed_values_function is the way to go, nice touch with the interface